pipelines/skylab/optimus/Optimus.changelog.md

```diff
@@ -1,7 +1,9 @@
 
 # 5.8.3
-2023-06-15 (Date of Last Commit)
+2023-06-23 (Date of Last Commit)
 
+* Updated STARsolo version to v2.7.10b for the StarsoloFastq task
+* Updated STARsolo argument for counting mode to GeneFull_Ex50pAS 
```

Comment:
Hey Liz - would you mind updating the docs where the "GeneFull" argument is mentioned? I think it's only mentioned once in the [Optimus Overview doc](https://github.com/broadinstitute/warp/blob/develop/website/docs/Pipelines/Optimus_Pipeline/README.md) in the tasks and tools section.

Reply:
Thanks @kayleemathews . I updated on this PR: https://github.com/broadinstitute/warp/pull/1010/files

Reply:
Thanks Liz!